RELEASE.md

```diff
@@ -1,3 +1,10 @@
+# Release 1.0.4
+
+* Certification for the 100.8.0 release of the ArcGIS Runtime SDK for iOS.
+
+Note:
```

Comment:
I think this Note might be better served in the README rather than the Release notes. What do you think?